src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -1006,6 +1016,59 @@ bool ResumptionDataDB::SelectSubscriptionsData(
   return true;
 }
 
+bool ResumptionDataDB::SelectUserLocationData(
+    const std::string& policy_app_id,
+    const std::string& device_id,
+    smart_objects::SmartObject& saved_app) const {
+  SDL_LOG_AUTO_TRACE();
+  using namespace app_mngr;
+  using namespace smart_objects;
+  uint32_t count_item = 0;
+  if (!SelectCountFromArray(
+          count_item, kSelectCountUserLocation, policy_app_id, device_id)) {
+    return false;
+  }
+
+  saved_app[strings::user_location] = SmartObject(SmartType_Map);
```

Comment:
@VadymLuchko I believe this line can be removed at all

Reply:
fixed in bf0ce24216a70351394e14311bafcfac997606f6